lms/static/sass/views/_homepage.scss

```diff
@@ -104,7 +104,6 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
           margin: ($baseline*0.25) 0 ($baseline*1.75) 0;
           padding: 0 ($baseline*0.75);
           height: $course-title-height;
-          color: $link-color;
```

Comment:
I just confirmed locally that this causes a display issue.

![image](https://cloud.githubusercontent.com/assets/163763/9766135/65dc5220-56e5-11e5-9e01-8d76fee6164b.png)

Please make sure this works as it does on master now.